app/src/main/res/drawable/ic_chevron_down_6.xml

```diff
@@ -8,6 +8,6 @@
     android:viewportWidth="6"
     android:viewportHeight="6">
     <path
-        android:fillColor="?attr/textPrimary"
+        android:fillColor="#000000"
```

Comment:
is it alright that the color will now be black for light/dark?

Reply:
This looks like an unintended change to me. Should we need to change the icon color, we should only do that through the iconTint in the ImageView

Reply:
These colour changes are not mistakes.  Search engines takes a bitmap when created.  At the moment we can't change the colour after we set the icon.  UX approved the solution that we can use this temporarily until a better solution is found.

Reply:
Let's talk about this tomorrow to make sure we are synced. At the very least, I don't think we necessarily have to make the UX changes in this PR.

Reply:
As agreed for now I will apply the tint to ic_chevron_down_6 and use photonBlack here.  Thanks